src/reflect/scala/reflect/internal/Definitions.scala

```diff
@@ -592,10 +592,10 @@ trait Definitions extends api.StandardDefinitions {
       (sym.tpe member nme.main).alternatives exists isJavaMainMethod
 
     class VarArityClass(name: String, maxArity: Int, countFrom: Int = 0, init: Option[ClassSymbol] = None) extends VarArityClassApi {
-      private val offset = countFrom - init.size
+      private[this] val offset = countFrom - init.size
       private def isDefinedAt(i: Int) = i < seq.length + offset && i >= offset
       val seq: IndexedSeq[ClassSymbol] = (init ++: countFrom.to(maxArity).map { i => getRequiredClass("scala." + name + i) }).toVector
-      private val symSet = new SymbolSet(seq.toList)
+      private[this] val symSet = new SymbolSet(seq.toList)
```

Comment:
@xuwei-k Yoshida-san, do you agree that it seems weird that SymbolSet takes a List in its constructor instead of an Iterable, given what it uses it for: https://github.com/scala/scala/blob/af5a0c61dda5701dfc516052293741865c18efcc/src/reflect/scala/reflect/internal/Definitions.scala#L155